io.catenax.return_request/2.0.0/ReturnRequest.ttl

```diff
@@ -0,0 +1,245 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.return_request:2.0.0#> .
+@prefix ext-quantity: <urn:samm:io.catenax.shared.quantity:1.0.0#> .
+
+:ReturnRequest a samm:Aspect ;
+   samm:preferredName "return request"@en ;
+   samm:description "Aspect to indicate if there is a return request for a product."@en ;
+   samm:properties ( :needsReturn :returnConditions :requestDate :latestReturnDate :productConditions :title :marketplaceProduct :quantity :unitOfMeasure :validityPeriod :isRecall :identification :productionDate :deliveryAddress :areaOfUsage :productGroup :arrivalDateAtCustomer ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:needsReturn a samm:Property ;
+   samm:preferredName "needs return"@en ;
+   samm:description "Describes the fact whether a vehicle, assembly or part needs to be returned to requestor."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:returnConditions a samm:Property ;
+   samm:preferredName "return conditions"@en ;
+   samm:description "Describes WHY a vehicle, assembly or part needs to be returned to the requestor and under which conditions."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Wishes to buy" .
+
+:requestDate a samm:Property ;
+   samm:preferredName "request date"@en ;
+   samm:description "Describes date when the return request was placed."@en ;
+   samm:characteristic :DateTimestamp ;
+   samm:exampleValue "2022-01-01"^^xsd:date .
+
+:latestReturnDate a samm:Property ;
+   samm:preferredName "latest return date"@en ;
+   samm:description "Describes until when the return request is valid."@en ;
+   samm:characteristic :DateTimestamp ;
+   samm:exampleValue "2025-01-01"^^xsd:date .
+
+:productConditions a samm:Property ;
+   samm:preferredName "product conditions"@en ;
+   samm:description "textual description of the product conditions"@en ;
+   samm:characteristic :ProductConditionsCharacteristic .
+
+:title a samm:Property ;
+   samm:preferredName "title"@en ;
+   samm:description "The request name to be used by the buyer to navigate through the list of demand requests."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "BMW 318i Brake caliper" .
+
+:marketplaceProduct a samm:Property ;
+   samm:preferredName "marketplace product"@en ;
+   samm:description "The description of the product within the marketplace that might differ from the digital twin."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "BMW 318i E46 Coupe Bj.2000 Bremssattel hinten links" .
+
+:quantity a samm:Property ;
+   samm:preferredName "quantity"@en ;
+   samm:description "The desired quantity of the on the marketplace offered product in SKU."@en ;
+   samm:characteristic ext-quantity:ItemQuantityCharacteristic .
+
+:unitOfMeasure a samm:Property ;
```

Comment:
this is redundant, as quantity (the shared aspect) also refers to a value and unit